packages/connect-iframe/webpack/prod.webpack.config.ts

```diff
@@ -40,6 +40,24 @@ const config: webpack.Configuration = {
                     filename: './workers/sessions-background-sharedworker.[contenthash].js',
                 },
             },
+            {
+                test: /sharedLoggerWorker.ts/i,
+                use: [
+                    {
+                        loader: 'worker-loader',
+                        options: {
+                            worker: 'SharedWorker',
+                            filename: './workers/shared-logger-worker.js',
+                        },
+                    },
+                    {
+                        loader: 'babel-loader',
+                        options: {
+                            presets: ['@babel/preset-typescript'],
+                        },
+                    },
+                ],
```

Comment:
btw what is the reason for adding this?

Reply:
It is necessary so the worker can be in typescript. The other workers in this file do not need it because they are pre transpiled in blockchain-link. At least that is my understanding. And it works.